src/main/resources/db/changelog/changes/v1.0/update-instance-authority-linking-rules.sql

```diff
@@ -0,0 +1,193 @@
+INSERT INTO linking_rules (linking_pair_type, jsonb)
```

Comment:
Shouldn't it be update?
Won't it throw "unique constraint violation" (linking_pair_type is a primary key)?

Reply:
No, we have:
`ON CONFLICT (linking_pair_type) DO UPDATE SET jsonb = EXCLUDED.jsonb;`

Reply:
replaced with updated